### PR TITLE
PLAT-1511: Update component events documentation

### PR DESCRIPTION
**Pull request by chavlji**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1511)

### CODE REVIEW

**Review by mihaelpalfi — APPROVED** [review]
Nice job Jan. Now its much more clear which events/methods/slots we have on the components. Just mention on the daily what and where we have to add to every new component that this info table get populated.

NIT: Only thing is maybe the styling of the methods table.  its a little different than for events and slots. And spacing between them.

**Review by f-jauregui — COMMENTED** [review]
This looks awesome, Jan!
Just one thing. Can we do the Methods description table look like the Events one? Or do we have another story to fix that?

Captura de Pantalla 2021-03-17 a la(s) 11 31 45

**Comment by chavlji** [comment]
@mihaelpalfi  @f-jauregui  changed it to look like this:
Screenshot 2021-03-17 at 15 44 52

**Review by f-jauregui — APPROVED** [review]
Awesome work, Jan! 🥇

File: src/stories/components/docs-table/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { JsonDocsComponent, JsonDocsEvent, JsonDocsTag } from '@stencil/core/internal/stencil-public-docs';
+
+function docTagToCustomEvent(docTag: JsonDocsTag): JsonDocsEvent {
+  const params = docTag.text.split('|');
```
Comment (homer0) [inline src/stories/components/docs-table/helpers.ts:4]: Same as the other :P

**Comment by homer0** [comment]
Yes, this was already merged; we agreed with @chavlji I would send the review anyways so he can have the comments in mind.

**Pull request merged**